### PR TITLE
fix transaction

### PR DESCRIPTION
**Pull request by luongdm-3080**
## Related Tickets
- [#50712](https://edu-redmine.sun-asterisk.vn/issues/50712)

## WHAT (optional)
- Change view form transaction.
- Search view time page

## HOW
- I edit html.erb file transaction.

## Evidence (Screenshot or Video)
Screenshot from 2022-07-20 09-45-15
Screenshot from 2022-07-20 09-44-43

### CODE REVIEW

**Comment by luongdm-3080** [comment]
ready

File: app/views/transactions/_form_transaction.html.erb [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
-    <%= f.select :category_id, options_from_collection_for_select(load_categories, :id, :name) %>
+    <%= f.select :category_id, options_from_collection_for_select(load_categories, :id, :name, selected: object.category_id || nil) %>
   </div>
   <div class="field">
     <%= f.label :transaction_date, t("transaction_date") %>
     <%= f.datetime_field :transaction_date, class: "form-control col-sm-3", min: "2014-05-20T00:00:00", value: @time_now %>
   </div>
   <%= f.label :total, t("total_label") %>
-  <%= f.text_field :total, class: "form-control" %>
+  <%= f.number_field :total, oninput: "validity.valid||(value='');", class: "form-control" %>
```
Comment (huongnt-2545) [inline app/views/transactions/_form_transaction.html.erb]: `oninput: "validity.valid||(value='');"` -> cho vô file js xử lý nhá

File: app/views/transactions/chart_analysis.html.erb [hunk 2]
```diff
@@ -1,3 +1,7 @@
+<div class="date-group chart-date">
+  <span class="date-start"> <%= l(@start_day.to_time, format: :covert) %></span>
```
Comment (huongnt-2545) [inline app/views/transactions/chart_analysis.html.erb]:
```suggestion
  <span class="date-start"> <%= l @start_day.to_time, format: :covert  %></span>
```

**Comment by luongdm-3080** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**